docs/command/project.soy

```diff
@@ -65,6 +65,26 @@
 {/call}
 </ul>
 
+<h2>Uncommon Parameters</h2>
```

Comment:
why do we need this? Looks like we don't use h2 in other place. I'd just add these parameters without putting them into a separate category